src/Uno.Toolkit.UI/Controls/Card/Card.Properties.cs

```diff
@@ -14,158 +14,264 @@
+			typeof(object),
+			typeof(Card),
+			new PropertyMetadata(default(object)));
 
-#region IconsContent and IconsContentTemplate
 		public object IconsContent
 		{
-			get { return (object)GetValue(IconsContentProperty); }
-			set { SetValue(IconsContentProperty, value); }
+			get => (object)GetValue(IconsContentProperty);
+			set => SetValue(IconsContentProperty, value);
 		}
 
-		public static readonly DependencyProperty IconsContentProperty =
-			DependencyProperty.Register("IconsContent", typeof(object), typeof(Card), new PropertyMetadata(null));
+		#endregion
+		#region DependencyProperty: IconsContentTemplate
+
+		public static DependencyProperty IconsContentTemplateProperty { get; } = DependencyProperty.Register(
+			nameof(IconsContentTemplate),
+			typeof(DataTemplate),
+			typeof(Card),
+			new PropertyMetadata(default(DataTemplate)));
 
 		public DataTemplate IconsContentTemplate
 		{
-			get { return (DataTemplate)GetValue(IconsContentTemplateProperty); }
-			set { SetValue(IconsContentTemplateProperty, value); }
+			get => (DataTemplate)GetValue(IconsContentTemplateProperty);
+			set => SetValue(IconsContentTemplateProperty, value);
 		}
 
-		public static readonly DependencyProperty IconsContentTemplateProperty =
-			DependencyProperty.Register("IconsContentTemplate", typeof(DataTemplate), typeof(Card), new PropertyMetadata(null));
-#endregion
+		#endregion
+
+		#region DependencyProperty: Elevation
+
+		public static DependencyProperty ElevationProperty { get; } = DependencyProperty.Register(
+			nameof(Elevation),
+			typeof(double),
+			typeof(Card),
+			new PropertyMetadata(default(double)));
 
-#region Elevation
 		public
 #if __ANDROID__
 			new
 #endif
 			double Elevation
 		{
-			get { return (double)GetValue(ElevationProperty); }
-			set { SetValue(ElevationProperty, value); }
+			get => (double)GetValue(ElevationProperty);
+			set => SetValue(ElevationProperty, value);
 		}
 
-		public static readonly DependencyProperty ElevationProperty =
-			DependencyProperty.Register("Elevation", typeof(double), typeof(Card), new PropertyMetadata(0));
-#endregion
+		#endregion
+		#region DependencyProperty: ShadowColor = DefaultShadowColor
+
+		public static DependencyProperty ShadowColorProperty { get; } = DependencyProperty.Register(
+			nameof(ShadowColor),
+			typeof(Windows.UI.Color),
+			typeof(Card),
+			new PropertyMetadata(DefaultShadowColor));
 
-#region ShadowColor
 		public Windows.UI.Color ShadowColor
 		{
-			get { return (Windows.UI.Color)GetValue(ShadowColorProperty); }
-			set { SetValue(ShadowColorProperty, value); }
+			get => (Windows.UI.Color)GetValue(ShadowColorProperty);
+			set => SetValue(ShadowColorProperty, value);
+		}
+
+		#endregion
+		#region DependencyProperty: IsClickable = true
+
+		public static DependencyProperty IsClickableProperty { get; } = DependencyProperty.Register(
+			nameof(IsClickable),
+			typeof(bool),
+			typeof(Card),
+			new PropertyMetadata(true));
+
+		/// <summary>
```

Comment:
If we had a summary here, can we had also a summary for the other properties, please?

Reply:
done